File: Iniciante/Pagina 03/1099/1099.py
contador = 0
casos = int(input())

for i in range(casos):
    a, b = map(int, input().split())

    if a > b:
        fim = a
        inicio = b + 1
    else:
        fim = b
        inicio = a + 1
    
    for inicio in range(inicio, fim):
        if (inicio % 2 != 0):
            contador += inicio

    print(contador)
    contador = 0

File: Iniciante/Pagina 03/1099/1099.c
#include <stdio.h>

int main() {
    int casos, contador = 0, a, b, fim, inicio;
    int i;

    scanf("%d", &casos);

    for (i = 0; i < casos; i++) {
        scanf("%d %d", &a, &b);
        
        if (a > b) {
            fim = a;
            inicio = b + 1;
        } else {
            fim = b;
            inicio = a + 1;
        }

        for (;inicio < fim; inicio++) {
            if (inicio % 2 != 0) {
                contador += inicio;
            }
        }

        printf("%d\n", contador);
        contador = 0;
    }

}